pkg/terraformer/terraformer.go

```diff
@@ -517,3 +507,8 @@ func (t *terraformer) cleanupJob(ctx context.Context, jobPodList *corev1.PodList
 	}
 	return nil
 }
+
+func (t *terraformer) computePodName(command string) string {
+	suffix := utils.ComputeSHA256Hex([]byte(time.Now().String()))[:5]
```

Comment:
Can we remove this on our side? Instead, can we simply use `.metadata.generateName`? We will find the pods anyways via labels.

Reply:
Yes, switched to `.metadata.generateName`. Nice suggestion.